app/views/Licenses.tsx

```diff
@@ -33,6 +32,8 @@ export const LicensesScreen = ({
   const { t } = useTranslation();
   const { legalHeader } = useAssets();
 
+  const legalHeaderText: string = legalHeader as string;
```

Comment:
the `Asset` type is a bit wonky and was causing some issues.  Text content is a string and not an Asset so it seems like we could refactor useAssets() to return Assets only hand handle the translations is a different way than the assets as to avoid some of these oddities in they type system.